### ISSUE TITLE
Allow multi-selection when delete multiple variables from a table

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#162]
```
for example, right now it is a real pain to “delete” multiple variables 
from a table. the UI is all set up for just a single variable. two things are 
wrong for when you have multiple variables: 
 - the variable list goes back to the top and does not stay with the last selected variable (since that one has disappeared from the list,  I assume the default is to go back to the top), that means you have  to scroll all the way back down to where you were - there is no way to select multiple variables (e.g., with shift-click or
  command-click).
```

Original issue reported on code.google.com by `lixun...@gmail.com` on 29 Jun 2015 at 11:43

**Comment by jkoschinsky** [comment]
Since you can now delete more than one variable, let's rename this from "Delete Variable" to "Delete Variable(s)"

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.87 (OSX Yosemite, 10.10.5 and Win 7 Professional (64-bit)).
